src/libstore/store-api.cc

```diff
@@ -783,6 +783,24 @@ void copyStorePath(ref<Store> srcStore, ref<Store> dstStore,
 }
 
 
+std::map<StorePath, StorePath> copyPaths(ref<Store> srcStore, ref<Store> dstStore, const RealisedPath::Set & paths,
+    RepairFlag repair, CheckSigsFlag checkSigs, SubstituteFlag substitute)
+{
```

Comment:
`settings.isExperimentalFeatureEnabled("ca-derivations")` somewhere in here?

Reply:
For the local side

Reply:
I think in practice this always hold (because the `RealisedPathsCommand` only uses `OpaquePath`s if the flag isn't set. But I've added a check here too in 00a30ea just for security